test/NewsService.Business.UnitTests/GetNewsByIdCommandTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using LT.DigitalOffice.Kernel.Exceptions;
+using LT.DigitalOffice.NewsService.Business.Interfaces;
+using LT.DigitalOffice.NewsService.Data.Interfaces;
+using LT.DigitalOffice.NewsService.Mappers.ResponsesMappers.Interfaces;
+using LT.DigitalOffice.NewsService.Models.Db;
+using LT.DigitalOffice.NewsService.Models.Dto.ModelResponse;
+using LT.DigitalOffice.NewsService.Models.Dto.Models;
+using LT.DigitalOffice.UnitTestKernel;
+using Moq;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.NewsService.Business.UnitTests
+{
+    public class GetNewsByIdCommandTests
+    {
+        private IGetNewsByIdCommand _getNewsInfoByIdCommand;
+        private Mock<INewsRepository> _repositoryMock;
+        private Mock<INewsResponseMapper> _mapperMock;
+
+        private Guid newsId;
+        private NewsResponse newsresponse;
+        private DbNews dbNews;
+        private DbNews badNews;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _repositoryMock = new Mock<INewsRepository>();
+            _mapperMock = new Mock<INewsResponseMapper>();
+            _getNewsInfoByIdCommand = new GetNewsByIdCommand(_repositoryMock.Object, _mapperMock.Object);
+
+            newsId = Guid.NewGuid();
+            newsresponse = new NewsResponse { Id = newsId };
+            dbNews = new DbNews { Id = newsId };
+            badNews = new DbNews();
+
+            _repositoryMock.Setup(repository => repository.GetNewsInfoById(newsId)).Returns(dbNews).Verifiable();
```

Comment:
Use one code-style in all files, in NewsResponseMapperTests.cs you using a line for each method, but here you are using one-line-style

Reply:
done